src/components/atoms/List/List.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { MemoryRouter } from 'react-router-dom';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import List, { IProps } from './List';
+import listItemIcon from '../../../assets/listItemIcon.svg';
```

Comment:
We can use absolute path. Do not need to use convention like `../../`

Reply:
gonna use it next time, thanks for a reminder!